.github/workflows/docs-generator.yml

```
@@ -0,0 +1,151 @@
+
+    strategy:
+      matrix:
+        python: [3.12]
+        node: [20]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
+
+      - name: Setup pnpm
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 35: could not find expected ':'

(syntax-check)